### ISSUE TITLE
Ruleset listing endpoint: Add versions parameter

### ISSUE THREAD

**Issue by asdine** [heetch/regula#23]
The `GET /ruletsets/<prefix>?list&versions` endpoint must return all the ruleset versions sorted by path and versions ascending
The `versions` option is a boolean.

The payload must look like this:

```json
{
  "revision": "2087",
  "rulesets": [
    {
      "path": "a/b/c",
       "ruleset": {},
      "version": "aa"
    },
    {
      "path": "a/b/c",
       "ruleset": {},
      "version": "bb"
    },
    {
       "path": "d/e/f",
       "ruleset": {},
      "version": "ac"
    }
  ]
}
```

**Comment by YassineE** [comment]
The payload will look like this instead:
```
{
  "revision": "2087",
  "rulesets": [
    {
      "path": "a/b/c",
      "version": "aa"
    },
    {
      "path": "a/b/c",
      "version": "bb"
    },
    {
      "path": "d/e/f",
      "version": "ac"
    }
  ]
}
```
Without the `"ruleset"` key.

**Comment by asdine** [comment]
Sorry, it was a bad copy paste from the #43 issue. I updated the issue: It is the same payload as the normal list endpoint, the difference is that now we return all the rulesets versions, not just the latest

**Comment by YassineE** [comment]
What should be the behavior if the `versions` parameter is passed next to the `paths` one?

**Comment by asdine** [comment]
I'd say we return a 400

**Comment by YassineE** [comment]
Fixed in #60.
